e2e/tests/transfer/forwarding_test.go

```diff
@@ -36,16 +36,24 @@ func (s *TransferForwardingTestSuite) TestForwarding_WithLastChainBeingICS20v1_S
 	ctx := context.TODO()
 	t := s.T()
 
-	relayer, chains := s.GetRelayer(), s.GetAllChains()
+	testName := t.Name()
+	relayer := s.SetupDefaultPath(testName)
+
+	chains := s.GetAllChains()
 
 	chainA, chainB, chainC := chains[0], chains[1], chains[2]
 
-	channelAtoB := s.GetChainAChannel()
+	channelAtoB := s.GetChainAChannelForTest(testName)
+
+	s.Require().Len(s.GetChannelsForTest(chainA, testName), 1, "expected one channel on chain A")
+	s.Require().Len(s.GetChannelsForTest(chainB, testName), 2, "expected two channels on chain B")
+	s.Require().Len(s.GetChannelsForTest(chainC, testName), 1, "expected one channel on chain C")
```

Comment:
added some additional checks here since this was failing due to not correctly building up the channel mapping correctly in `CreatePath`